packages/axios-extension/src/abap/message.ts

```diff
@@ -38,6 +38,22 @@ export interface ErrorMessage {
     };
 }
 
+const logLevel = (severity: string, msg: string, log: Logger, error = false): void => {
```

Comment:
Could we use function syntax `function logLevel(severity ...)` since we use it in all other functions in this file.
And then, please add a JSDod comment

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/929/files